### DISCUSSION TITLE
检测到您的页面展示可能受到浏览器插件影响，建议您将当前页面加入插件白名单，以保障您的浏览体验～

### DISCUSSION THREAD

**Discussion by power721** [the1812/Bilibili-Evolved#3989] [功能建议 / Ideas]
Chrome安装了Adblock插件，首页显示
检测到您的页面展示可能受到浏览器插件影响，建议您将当前页面加入插件白名单，以保障您的浏览体验～
![image](https://user-images.githubusercontent.com/2384040/218290253-f26b7cea-e7d4-402c-9591-c1e1c4142607.png)

希望能去掉。

**Comment by the1812** [comment]
#3719

**Comment by RingingVolcano** [comment]
evolved本身就有屏蔽广告的功能 可以在abp设置里对B站禁用生效

**Reply by Nyte-BK201** [reply]
自定义顶栏提示使用adb屏蔽原生iframe以提高性能，adb不对b站生效的话，有什么替代方案吗？

**Reply by Nyte-BK201** [reply]
:) div="adblock-tips"并不是一个anti adblocker，所以我直接手动屏蔽了
